common/source/docs/common-pocketbeagle-2.rst

```diff
@@ -0,0 +1,291 @@
+
+.. note::
+
+   For now, the only supported version is Rev A0 shipped with AM6232 (dual-core A53, no GPU).
+
+.. image:: ../../../images/pocketbeagle-2-front.jpg
+    :target: ../_images/pocketbeagle-2-front.jpg
+    :width: 800 px
+
+.. image:: ../../../images/pocketbeagle-2-back.jpg
+    :target: ../_images/pocketbeagle-2-back.jpg
+    :width: 800 px
+
+Specifications
+==============
+
+-  **Processor**
+
+   -  Texas Instruments AM6232 SoC
+   -  Multicore 64-bit Arm Cortex-A53 microprocessor subsystem at up to 1.4 GHz
+   -  Each A53 Core has 32KB L1 DCache with SECDED ECC and 32KB L1 ICache with Parity protection
+   -  Single-core Arm® Cortex®-M4F MCU at up to 400MHz
+   -  Dual-core Programmable Real-Time Unit Subystem (PRUSS) running up to 333 MHz
+   -  512MB LPDDR4 3200MHz
+   -  4GB 8-bit eMMC flash
+
+-  **OS**
+
+   -  Linux (Debian)
+
+.. note::
+
+   For now, the only supported version is Rev A0 shipped with AM6232 (dual-core A53, no GPU).
+
+
+
+DIY Cape
+========
+The board and all information about the board can be found in the github `here. <https://github.com/juvinski/pbboard>`__
+
+
+.. image:: ../../../images/pocketbeagle-2-cape-top.jpg
+    :target: ../_images/pocketbeagle-2-cape-top.jpg
+    :width: 800 px
+
+.. image:: ../../../images/pocketbeagle-2-cape-components.jpg
+    :target: ../_images/pocketbeagle-2-cape-components.jpg
+    :width: 800 px
+
+-  **Schematic**
+
+The schematic can be found in this `link. <https://github.com/juvinski/pbboard/blob/main/Schematic/PocketPilot2.pdf>`__
+
+-  **Cape Components**
+
+   -  1 IMU `GY-912 <https://www.ebay.com/itm/166809278501>`__
+   -  4 Resistors 1KΩ
+   -  3 Led Red, Blue and Green color 
+   -  1 BC548 NPN Transistor
+   -  1 I2C Digital `Power Module/Monitor <https://ardupilot.org/copter/docs/common-powermodule-landingpage.html#i2c-power-monitor>`__
+   -  6 Pin header 1 x 5 straight
+   -  3 Pin header 1 x 4 straight
+   -  1 Pin header 1 x 6 straight
+   -  6 Pin header 1 x 3 straight or 90°
+   -  1 Pin header 1 x 2 straight
+   -  1 Pin header female 1 x 8 - optional
+
+-  **Board Details**
+
+   -  4 UARTs - One have the I2C to be used with a GPS + Compass
+   -  2 I2Cs - One is under the Power header
+   -  4 GPIO pins with GND
+   -  6 Outputs
+   -  Buzzer output for an Active buzzer
+
+
+Quick Start
+===========
+
+Preparing SD Card with linux for PB2
+------------------------------------
+
+
+Download the linux image from `https://files.beagle.cc/file/beagleboard-public-2021/images/pocketbeagle2-debian-13-base-v6.12-arm64-2025-09-05-8gb.img.xz <https://files.beagle.cc/file/beagleboard-public-2021/images/pocketbeagle2-debian-13-base-v6.12-arm64-2025-09-05-8gb.img.xz>`__ used in this example).
+You can download images from from https://www.beagleboard.org/distros or you can use the `BeagleBoard Imaging Utility. <https://beagleboard.github.io/bb-imager-rs/>`__
```

Comment:
The URL https://www.beagleboard.org/bb-imager should have long-term links to the supported BeagleBoard Imaging Utility.